h5py/h5d.pyx

```diff
@@ -395,3 +395,43 @@ cdef class DatasetID(ObjectID):
             """
             H5Drefresh(self.id)
 
+
+    IF HDF5_VERSION >= (1, 8, 11):
+
+        def write_direct_chunk(self, offsets, data, filter_mask=0, PropID dxpl=None):
+            """
+            Direct chunk write to dataset
```

Comment:
See other docstrings in the low-level interface for an example of what's expected here.